README.md

```diff
@@ -170,4 +170,10 @@ For snow cover fraction(scf), IMS grib2 files are supported with `ims_scf2ioda.p
 Usage: ims_scf2ioda.py -i input_ims_file.grib2 -o output_ioda_file.nc -m maskout
 ```
 For -i you can specify an input file and the converter will write it to one output file. For maskout option (-m) default/maskout, default means to keep all missing values and maskout means to not write out missing values.
-    
+
```

Comment:
It would be good to add instructions here explaining the dependency on the python pygrib module. And instructions how to enable the testing of these scripts when the user has a pygrib module available (ie, add -DUSE_PYGRIB=1 to the ecbuild command line). Thanks

Reply:
I added the sentences below:
It should be noted that both ims_scf2ioda.py and afwa_snod2ioda.py are depending on the python pygrib module. To enable the testing of these two scripts when the user has a pygrib module available, during the ecbuild process, please add -DUSE_PYGRIB=True to the ecbuild command line.